src/util.ts

```diff
@@ -1,27 +1,33 @@
-/* eslint-disable @typescript-eslint/no-var-requires */
-/* eslint-disable @typescript-eslint/explicit-function-return-type */
-const crypto = require('crypto');
-const database = require('./database');
-const logger = require('./logger');
-const grpc = require('nice-grpc');
-const config = require('../config.json');
-const { SETTINGS } = require('./models/settings');
-const { CONTENT } = require('./models/content');
-const { NOTIFICATION } = require('./models/notifications');
-const { COMMUNITY } = require('./models/communities');
-const { AccountDefinition } = require('@pretendonetwork/grpc/account/account_service');
-const { FriendsDefinition } = require('@pretendonetwork/grpc/friends/friends_service');
-const { APIDefinition } = require('@pretendonetwork/grpc/api/api_service');
-const translations = require('./translations');
-const HashMap = require('hashmap');
-const TGA = require('tga');
-const pako = require('pako');
-const PNG = require('pngjs').PNG;
-const bmp = require('bmp-js');
-const aws = require('aws-sdk');
-const crc32 = require('crc/crc32');
-const communityMap = new HashMap();
-const userMap = new HashMap();
+import aws from 'aws-sdk';
+import bmp from 'bmp-js';
+import crc32 from 'crc/crc32';
+import crypto from 'crypto';
+import HashMap from 'hashmap';
+import grpc from 'nice-grpc';
+import pako from 'pako';
+import { PNG } from 'pngjs';
+import sharp from 'sharp';
+import TGA from 'tga';
+import { AccountDefinition } from '@pretendonetwork/grpc/account/account_service';
+import { APIDefinition } from '@pretendonetwork/grpc/api/api_service';
+import { FriendsDefinition } from '@pretendonetwork/grpc/friends/friends_service';
+import logger from '@/logger';
+import * as database from '@/database';
+import { COMMUNITY } from '@/models/communities';
+import { CONTENT } from '@/models/content';
+import { NOTIFICATION } from '@/models/notifications';
+import { SETTINGS } from '@/models/settings';
+import translations from '@/translations';
+import type { LoginResponse as ApiLoginResponse } from '@pretendonetwork/grpc/api/login_rpc';
+import type { FriendRequest } from '@pretendonetwork/grpc/friends/friend_request';
+import type { ParamPack } from '@/types/common/param-pack';
+import type { Token } from '@/types/common/token';
+import type { User } from '@/types/common/user';
+import type { INotification } from '@/types/mongoose/notifications';
+import config from '../config.json';
+
+const communityMap = new HashMap<string, string>();
+const userMap = new HashMap<number, string>();
 
 const { ip: friendsIP, port: friendsPort, api_key: friendsKey } = config.grpc.friends;
 const friendsChannel = grpc.createChannel(`${friendsIP}:${friendsPort}`);
```

Comment:
```suggestion
const friendsChannel = createChannel(`${friendsIP}:${friendsPort}`);
```
I won't add this for each instance, but changing the imports to not be on the grpc object means we need to change the calls throughout this file. There might be a better way to do this but ¯\_(ツ)_/¯